Wikipedia/Code/DescriptionEditViewController.swift

```diff
@@ -84,6 +84,9 @@ protocol DescriptionEditViewControllerDelegate: AnyObject {
         descriptionTextView.textContainerInset = .zero
         
         updateFonts()
+        if let articleURL = articleDescriptionController.article.url {
+            EditAttemptFunnel.shared.logInit(articleURL: articleURL)
```

Comment:
I notice `EditAttemptFunnel.shared.logInit(articleURL: articleURL)` is called in ArticleViewController+Editing in `showEditorForSectionOrTitleDescription`, with this you might see double `init` logs when going into the article description editor.